cmd/aspect/main.go

```diff
@@ -49,6 +50,12 @@ func main() {
 
 	cmd := root.NewDefaultRootCmd(pluginSystem)
 
+	// Run this command after all bazel verbs have been added to "cmd".
+	if err := flags.AddBazelFlags(cmd); err != nil {
+		fmt.Fprintln(os.Stderr, "Error:", err)
```

Comment:
this seems like strange error handling, is it intended?

Reply:
This is the main function. In this case, we just print the error to the user and exit non-zero.